Procfile

```diff
@@ -1,2 +1,3 @@
-vue: vue-cli-service build --mode development --watch
+vite: vite dev
+build: NODE_ENV=development vite build --mode development --watch
```

Comment:
Even though we're mostly using Vite modes now, we still need to specify `NODE_ENV=development`, because vite.config.js references `NODE_ENV`. `vite dev` specifies `NODE_ENV=development` automatically, which is why only this line has it.

Reply:
we can use [conditional config](https://vite.dev/config/#conditional-config] and remove NODE_ENV from here